Source/WebCore/Modules/cookie-store/CookieListItem.h

```diff
@@ -63,8 +63,8 @@ struct CookieListItem {
     String domain;
     String path;
     std::optional<DOMHighResTimeStamp> expires;
-    bool secure;
-    CookieSameSite sameSite;
+    bool secure { false };
+    CookieSameSite sameSite { CookieSameSite::Strict };
```

Comment:
I'm not sure that these changed should be made. The IDL files do not specify that we should have default values for `sameSite` and `secure`. But, if we do not have the default values, the tests that are newly passing with this patch will not pass. The reason seems to be that the tests do not specify a value for `sameSite` when creating the `CookieChangeEvent`. The JS bindings default initialize the `CookieListItems` (which don't have a default value for `sameSite`) and so `sameSite` gets some garbage value. This causes us to hit the `ASSERT(static_cast<size_t>(enumerationValue) < std::size(values));` assertion in `JSCookieSameSite.cpp:43`.  I'm not sure what the correct course of action is here. Should we use default values? Or alter the test? Or perhaps something else?

Reply:
Seems fine.